CMake/FindIBMMASS.cmake

```diff
@@ -2,37 +2,31 @@
 include(CheckCXXSourceCompiles)
 
 message(STATUS "Looking for IBM MASS libraries")
-# Finding and setting the MASS_INCLUDE_DIRECTORIES
-set(SUFFIXES include)
+
+# Finding MASS_INCLUDE
 find_path(
-  MASS_INCLUDE_DIRECTORIES name "mass.h"
```

Comment:
generally find modules set PACKAGE_INCLUDE_DIRECTORIES and PACKAGE_LIBRARIES.  However this shouldn't matter if we just use the targets from here on.

Reply:
Using target also simplify try_compiler. Quite nice to do it this way.

I found for include, I won't get much confused. We only need a path to the directory.
For library, I tend to make more strict choice mainly to distinguish the path the library and the library files themselves. LIBS or LIBRARIES must refer to the library files. `_DIRS` or ` _DIRECTORY` are needed for library directory.